TensorLib/Tensor.lean

```diff
@@ -0,0 +1,135 @@
+import Mathlib.Tactic
```

Comment:
You're supposed to use a copyright notice such as: 
/-
Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Foo Bar
-/

Reply:
Done

Reply:
Done